third_party/blink/renderer/core/brave_page_graph/graphml.cc

```diff
@@ -25,10 +25,14 @@ uint32_t graphml_index = 0;
 GraphMLAttr::GraphMLAttr(const GraphMLAttrForType for_value,
                          const String& name,
                          const GraphMLAttrType type)
-    : id_(++graphml_index), for_(for_value), name_(name), type_(type) {}
+    : id_(++graphml_index),
+      for_(for_value),
+      name_(name),
+      type_(type),
+      graphml_id_("d" + base::NumberToString(id_)) {}
 
 GraphMLId GraphMLAttr::GetGraphMLId() const {
```

Comment:
this now should return `const GraphMLId&` to not make unnecessary copies each call.